pkg/sync/sync_context.go

```diff
@@ -721,14 +723,17 @@ func (sc *syncContext) runTasks(tasks syncTasks, dryRun bool) runState {
 	{
 		var wg sync.WaitGroup
 		for _, task := range pruneTasks {
+			task := task
```

Comment:
👍

Reply:
This line is not needed as `task` is passed to the closure (line 737/742) and it uses `t` instead.

Reply:
I think that's been fixed with this commit.
https://github.com/argoproj/gitops-engine/ pull/51/commits/c5d445f642a1283a09aa0c 0056274e1e6fe17997